src/erc20/tokens.ts

```diff
@@ -67,6 +67,60 @@ const tokens = {
     address: "0xba100000625a3754423978a60c9317c58a424e3D",
     abi: ERC20Abi,
   },
+  usdt: {
+    symbol: "USDt",
```

Comment:
This is supposed to be USDT, I believe:
See: https://etherscan.io/token/0xdAC17F958D2ee523a2206206994597C13D831ec7

Reply:
Yeah. Typo error